azure-devops/azext_devops/dev/team/invoke.py

```diff
@@ -106,24 +106,29 @@ def invoke(area=None, resource=None,
                             media_type=media_type,
                             accept_media_type=accept_media_type,
                             content=request_body)
-
     logger.info('content type header')
     logger.info(response.headers.get("content-type"))
-    if 'json' in response.headers.get("content-type") and not out_file:
+    no_content = False
+
+    if not response.headers.get("content-type"):
+        logger.info('Content type header is None.')
+        no_content = True
+    elif 'json' in response.headers.get("content-type") and not out_file:
         return response.json()
 
-    if not out_file:
-        raise CLIError('Response is not json, you need to provide --out-file where it can be written')
+    if not no_content:
```

Comment:
Logic is correct. 

To be honest, took some time to review the combination of no_content and not. I felt it would be a bit easier if we can use something like isContentAvailable as it will make is easy to read and comprehend in review. 

I mean, instead of maintaining is content not available in response, we can use the variable to contain the value to represent if content is available. Just the other way.

Reply:
Done. Thanks!